src/components/FeatureView/FeatureView.js

```diff
@@ -84,7 +81,14 @@ export default function FeatureViewCard() {
 											checked={todo.isChecked}
 											onChange={() => checkTodo(index)}
 										/>
-										{todo.name}
+										<StyledDiv
+											variant="open-todonote"
+											onClick={() => {
+												openNote(index);
+											}}
+										>
+											{todo.name}
+										</StyledDiv>
```

Comment:
If you want your user to interact with an element like clicking on it you should use a button component, otherwise it is ignored when you use `Tab` to navigate your app. This is important for accessibility.